src/ngraph/op/fused/rnn_cell.cpp

```diff
@@ -73,7 +73,9 @@ op::RNNCell::RNNCell(const Output<Node>& X,
 
 bool op::RNNCell::visit_attributes(AttributeVisitor& visitor)
 {
-    return op::util::RNNCellBase::visit_attributes(visitor);
+    op::util::RNNCellBase::visit_attributes(visitor);
+    m_activation_f = get_activation_function(0);
```

Comment:
What is this for?

Reply:
The [python RNNCell test](https://aipg-jenkins-algo.intel.com/blue/organizations/jenkins/ngraph-python-merge/detail/ngraph-python-merge/15170/pipeline/69#step-72-log-435) failed on the CPU backend (the same test for the INTERPRETER passed).
I discovered that for RNNCell created by node factory, the activation function on CPU backend is always 'tanh' despite we passed 'sigmoid' in the `activations` vector attribute.

I just wanted to run the CI with this hot fix and I see that it works now.
But probably this problem required deeper investigation, not in this PR scope.
So I can revert this change and skip the python RNNCell test on the CPU backend.
Like I said for the INTERPRETER backend the CI passed anyway.